test/snapshot.js

```diff
@@ -1,33 +1,38 @@
-const fs = require("node:fs");
-const path = require("node:path");
+const fs = require("fs");
```

Comment:
please use `node:` namespace for imports

Reply:
ı used, thanks